frontend/src/stories/fixtures/activityLogList.json

```diff
@@ -5,12 +5,20 @@
     "timestamp": "2022-09-02T13:46:16.295020Z",
     "actorUser": 352,
     "effectedUser": 802,
+    "actorUserEmail": "faith.mofokeng@umuzi.org",
+    "effectedUserEmail": "faith.mofokeng@umuzi.org",
     "object1ContentTypeName": "git_real | pull request review",
     "object1Id": 53534,
     "object2ContentTypeName": "git_real | repository",
     "object2Id": 7250,
+    "object_1Summary": {
```

Comment:
are you sure it wouldn't be object1Summary?

Reply:
Hmm that is weird. It does do that but why?